Design/Instrument-2.0/Detector_Scan_NeXus_Saving.md

```diff
@@ -110,17 +111,20 @@ To resolve the differences between the NeXus proposal and Mantid one option is t
   * If the parent is scanning it should throw
 * The in memory representation is the same as before - the absolute position and rotation of all pixels are stored
 
-The performance impact of this is that every call to `DetectorInfo::position()` will have an extra branch, even for non-scanning workspaces.
+The performance impact of this is that every call to `DetectorInfo::setPosition()` and similar will have an extra branch, even for non-scanning workspaces.
 
 An alternative approach is to have something similar to `RectangularDetectors`. Here instead of storing the scan positions for each detector we would just store the position for the parent. Positions would need to computed on the fly to get the actual position for each detector. This would have the advantage of reducing memory use*, at the cost of more CPU work. This would be a step away from some of the Instrument 2.0 design principles.
 
 *Note that the memory overhead due to scanning is not the main issue for the ILL instruments, addressing the overhead from using histograms with a  single count (no ToF) is a higher priority.
 
-#### Outstanding Questions
+#### Outstanding Question
 
-* If components scan, are detectors still allowed to scan independently? 
+* If components scan, are pixels within that component still allowed to scan independently?
+  * Possible use cases for this would be tubes that might change in time, for example temperature, during a scan. The calibration might then be time dependent.
```

Comment:
I can't speak speak for other facilities, but tube calibration isn't performed like this at ISIS to my knowledge. I don't like the idea of a scan within a scan.